modules/http_push.c

```diff
@@ -81,6 +85,13 @@ static int nw_http_push_start_acquire_data(struct nodewatcher_module *module,
         free(client_certificate);
         free(client_key);
 
+        /* provide a buffer to store errors in */
```

Comment:
Please fix comments so they start with a capital letter and end with a dot (also below).